launchcontainer/static/js/src/launchcontainer_edit.js

```diff
@@ -1,25 +1,53 @@
 function LaunchContainerEditBlock(runtime, element) {
+
     console.log($('.xblock-save-button', element));
+
+    function notify(response) {
+        var $notificationEl = $('#notification-message'); 
+        var $editModal = $('.wrapper.wrapper-modal-window.wrapper-modal-window-edit-xblock');
+        var $editModalOverlay = $('.modal-window-overlay');
+        var $displayHeader = $('.xblock-header.xblock-header-launchcontainer .header-details');
+        var $editSuccessContent = "Your changes have been successfully applied. "
+                                  + "Please refresh and then click the button above to test.";
+        var $launcherNotification = $('#launcher_notification');
+
+        if (response.result === 'success') {
+          // class="ui-loading is-hidden"
+          
+          // Hide the modal in the Studio interface.
+//          $notificationEl.find('span').html('Your request was successful.');
+          $editModal.addClass('is-hidden');
+          $editModalOverlay.addClass('is-hidden');
+          $launcherNotification.text($editSuccessContent);
+          $launcherNotification.addClass('ui-state-highlight')
+                               .removeClass('ui-state-error')
+                               .removeClass('hide');
+          $('#launcher_form').removeClass('hide');
+        }
+        else {
+          $notificationEl.addClass('error').removeClass('hide').removeClass('is-hidden');
+          $notificationEl.html("It didn't work! "+response.result); 
+        }
+    }
+
     $('.save-button', element).bind('click', function() {
         var handlerUrl = runtime.handlerUrl(element, 'studio_submit');
         var data = {
-            'project': $('input[name=project]').val(),
-            'project_friendly': $('input[name=project_friendly]').val(),
+            'project': $('input#project_input').val(),
+            'project_friendly': $('input#project_friendly_input').val(),
+            'project_token': $('input#project_token_input').val(),
         };
-        runtime.notify('save', {state: 'start'});
-        $.post(handlerUrl, JSON.stringify(data)).done(function(response) {
-            if (response.result === 'success') {
-                runtime.notify('save', {state: 'end'});
-            }
-            else {
-                runtime.notify('save', {state: 'end'});
-                $('#error-message', element).html('Error: '+response.result);
-            }
+
+        $.post(
+            handlerUrl, 
+            JSON.stringify(data))
```

Comment:
Yes, exactly. But it's nothing major, https://api.jquery.com/jquery.post/ says either version is fine.